### DISCUSSION TITLE
Disable Bulk Actions

### DISCUSSION THREAD

**Discussion by MACscr** [filamentphp/filament#365] [Ideas]
There should be a way to hide/disable bulk actions when creating the table. There are plenty of occasions where I have no use for such a feature or don't want a particular role to have access to such a feature and it would be beneficial to not have them take up so much valuable screen real estate.

**Comment by tryoasnafi** [comment]
Hello there, I got a little trick to solve this issue, I just pass an empty array to `$table->bulkActions([])`
